src/AasxServerBlazor/Pages/Db.razor

```diff
@@ -1,905 +1,803 @@
+                    <br />
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">First 1000 as TABLE</span></b>
+                        <br />
+                    }
+                    <table class="table table-bordered table-sm" style="width:100%;word-wrap:break-word;word-break:break-all;">
+                        <thead>
+                            <tr>
+                                <th style="word-break:keep-all">AASX#</th>
+                                <th style="word-break:keep-all">AAS#</th>
+                                <th style="word-break:keep-all">SM#</th>
+                                <th style="word-break:keep-all">Id</th>
+                                <th style="word-break:keep-all">IdShort</th>
+                                <th style="word-break:keep-all">SemanticId</th>
+                                <th style="word-break:keep-all">Links</th>
+                                <th style="word-break:keep-all">API</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @for (int i = 0; i < data.Count && i < 1000; i++)
+                            {
+                                var smDB = data[i];
+                                string submodel64 = Base64UrlEncoder.Encode((smDB.Identifier));
+                                string link = AasxServer.Program.externalBlazor + "/submodels/" + submodel64;
+                                int smeCount = db.SMESets.Where(sme => sme.SMId == smDB.Id).Count();
+
+                                <tr>
+                                    <td style="word-break:keep-all">@smDB.AASXId</td>
+                                    <td style="word-break:keep-all">@smDB.AASId</td>
+                                    <td style="word-break:keep-all"><b>@smDB.Id</b></td>
+                                    <td>@smDB.Identifier</td>
+                                    <td>@smDB.IdShort</td>
+                                    <td>@smDB.SemanticId</td>
+                                    <td style="word-break:keep-all">
+                                        <a href="@(shorturl + "/aasx?aasxid=" + smDB.AASXId)" target="_blank">aasx </a>
+                                        <a href="@(shorturl + "/aas?aasid=" + smDB.AASId)" target="_blank">aas </a>
+                                        <a href="@(shorturl + "/sme?smid=" + smDB.Id)" target="_blank">sme:@(smeCount)</a>
+                                    </td>
+                                    <td><a href="@link" target="_blank">@link</a></td>
+                                </tr>
+                            }
+                        </tbody>
+                    </table>
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">All other as text</span></b>

```

Comment:
use the css for styling and don't style your elements in the file directly. Makes it harder to change later